src/transformers/models/llama/modeling_llama.py

```diff
@@ -1413,3 +1421,108 @@ def forward(
             hidden_states=transformer_outputs.hidden_states,
             attentions=transformer_outputs.attentions,
         )
+
+
+@add_start_docstrings(
+    """
+    The Llama 2 Model transformer with a span classification head on top for extractive question-answering tasks like
+SQuAD (a linear layer on top of the hidden-states output to compute `span start logits` and `span end logits`).
+    """,
+    LLAMA_START_DOCSTRING,
+)
+class LlamaForQuestionAnswering(LlamaPreTrainedModel):
```

Comment:
there should be some copied from here

Reply:
this still needs to be addressed 🤗

Reply:
Hey @ArthurZucker  Sorry but those copy statements are not working .. i tried copied statements of gptj and bloom but they all are different at some stages.

Reply:
@ArthurZucker  Hey thanks for assist .. I have given copied comment to `__init__` function but forward function is too different from other models because parameters are different so it didn't worked for that . I have tested it in goggle colab and working fine as expected and all testcases are passing.